packages/cli/src/verify.ts

```diff
@@ -105,7 +105,7 @@ export async function verify({
     );
 
     modules.map(({ name, prepareDeploy }) => {
-      const { address } = prepareDeploy(deployerAddress, []);
+      const { address } = prepareDeploy(deployerAddress, {});
```

Comment:
thoughts on defaulting the second arg in `prepareDeploy` so callers don't need to know the shape of the thing to pass in